Gemfile

```diff
@@ -13,7 +13,7 @@ group :preload do
   # AR extensions
   gem 'goldiloader'
   gem 'pagy'
-  gem 'audited'
+  gem 'audited', git: "https://github.com/yknx4/audited.git", branch: "patch-1" # https://github.com/collectiveidea/audited/pull/554
```

Comment:
seems a little shady to not wait for an official release

Reply:
changes are very minimal, so I'd hope this is fine 🤞 
https://github.com/collectiveidea/audited/pull/554/files

Reply:
got a real release now